### ISSUE TITLE
Very very low predict accuracy after train darknet19_448 model on imagenet 1k dataset

### ISSUE THREAD

**Issue by lqian** [pjreddie/darknet#439]
For practice purpose I train darket19_448 model on ImageNet 1k dataset via the command
`./darknet classifier train cfg/imagenet1k.data cfg/darknet19_448.cfg`

Since the darket project does not provide train list for imagenet 1k dataset. I write a python to generate the train list. the file content has lines like below.
`tail data/imagenet1k.train.list 
/home/link/ImageNet/train/JPEGImages/n15075141/n15075141_9408.JPEG
/home/link/ImageNet/train/JPEGImages/n15075141/n15075141_9439.JPEG
/home/link/ImageNet/train/JPEGImages/n15075141/n15075141_9453.JPEG
/home/link/ImageNet/train/JPEGImages/n15075141/n15075141_9461.JPEG
/home/link/ImageNet/train/JPEGImages/n15075141/n15075141_9488.JPEG
/home/link/ImageNet/train/JPEGImages/n15075141/n15075141_9596.JPEG
/home/link/ImageNet/train/JPEGImages/n15075141/n15075141_9628.JPEG
/home/link/ImageNet/train/JPEGImages/n15075141/n15075141_9725.JPEG
/home/link/ImageNet/train/JPEGImages/n15075141/n15075141_9729.JPEG`

After train 10 epochs, the model just has almost 18% top-1 accuracy and 41% accuracy. Also, the lost
always is greater than 0.9 in the whole training.
The accuray is very lower than pjreddie site declarative **I trained Darknet19 for 10 more epochs with a larger input image size, 448x448. This model performs significantly better but is slower since the whole image is larger.**

I'm wondering how many epochs the pre-trained darknet19_448 model has. 
Also, Have I made an mistake in the train list?

`6818: top 1: 0.177005, top 5: 0.406511
6819: top 1: 0.176979, top 5: 0.406452
6820: top 1: 0.177100, top 5: 0.406539
6821: top 1: 0.177221, top 5: 0.406626
6822: top 1: 0.177195, top 5: 0.406566
6823: top 1: 0.177169, top 5: 0.406506
6824: top 1: 0.177143, top 5: 0.406447
6825: top 1: 0.177117, top 5: 0.406387
6826: top 1: 0.177091, top 5: 0.406328
6827: top 1: 0.177211, top 5: 0.406415
6828: top 1: 0.177186, top 5: 0.406355`

**Comment by lqian** [comment]
update final training log

`Loaded: 0.000165 seconds`
`99996, 9.991: 0.953195, 0.954291 avg, 0.000000 rate, 1.819176 seconds, 12799488 images`
`Loaded: 0.000092 seconds`
`99997, 9.991: 0.948380, 0.953700 avg, 0.000000 rate, 1.937180 seconds, 12799616 images`
`Loaded: 0.000097 seconds`
`99998, 9.991: 0.954728, 0.953803 avg, 0.000000 rate, 1.952160 seconds, 12799744 images`
`Loaded: 0.000084 seconds`
`99999, 9.991: 0.965320, 0.954954 avg, 0.000000 rate, 2.022633 seconds, 12799872 images`
`Loaded: 0.000095 seconds`
`100000, 9.991: 0.936787, 0.953138 avg, 0.000000 rate, 2.037446 seconds, 12800000 images`
`Saving weights to backup//darknet19_448.backup`
`Saving weights to backup//darknet19_448.weights`

**Comment by ouyangkid** [comment]
try to use batch=1 during testing and validation
